devicon.json

```
@@ -7289,7 +7314,7 @@
             "programming"
         ],
         "versions": {
-            "svg": [ 
+            "svg": [
```

Let's keep changes only to `grpc` entry.
```suggestion
            "svg": [ 
```